corehq/motech/openmrs/tests/test_repeaters.py

```diff
@@ -74,7 +81,7 @@ def test_get_case_updates_for_followup(self):
         )
 
 
-class GetPatientByUuidTests(TestCase):
+class GetPatientByUuidTests(unittest.TestCase):
```

Comment:
(I figured it should be more explicit that these are unittest TestCases, not Django TestCases.)

Reply:
Any reason this uses ```unittest.TestCase``` and not ```SimpleTestCase```?

Reply:
I tried changing them from `SimpleTestCase` to `unittest.TestCase` and they seemed faster, so I left them that way. ... But it was more likely my imagination than that they were actually faster. It's also confusing because all our other tests extend Django test cases. And this change has given me merge conflicts. So when things quieten down in this file, I'll change them back to SimpleTestCases and try not to fix things that aren't broken. :upside_down_face: